packages/spelunker-web/.eslintrc

```diff
@@ -5,8 +5,14 @@
     "eslint-config-airbnb/rules/react-a11y"
   ],
   "rules": {
-    "jsx-a11y/anchor-is-valid": "never",
-    "react/prop-types": ["never"]
+    "jsx-a11y/anchor-is-valid": "off",
+    "react/button-has-type": "off",
+    "react/destructuring-assignment": "off",
+    "react/forbid-prop-types": "off",
+    "react/jsx-one-expression-per-line": "off",
+    "react/jsx-props-no-spreading": "off",
+    "react/jsx-wrap-multilines": "off",
+    "react/prop-types": ["off"]
```

Comment:
The two rules that changed from `never` are corrections. The additional rules set to `off` are from the updated ESLint deps, and were flagged off since they hit a bunch of cases. We can consider re-enabling them later!